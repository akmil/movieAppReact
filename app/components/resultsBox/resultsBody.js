import React from 'react';
import {ResultsBodyItem} from './resultsBodyItem';

export const ResultsBody = ({ movies = [], onMovieClick }) => {
    return (
        <div className='results-list-box'>
            <ul className='results-list'>
                {movies.map((movie, index) => (
                    <ResultsBodyItem key={index} {...movie} onClick={() => onMovieClick(index)} />
                ))}
            </ul>
        </div>
    )
};