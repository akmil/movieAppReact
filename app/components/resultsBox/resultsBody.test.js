import React from 'react';
// import renderer from 'react-test-renderer';
import {ResultsBody} from "./resultsBody";
import { shallow } from 'enzyme'

describe('ResultsBody', function () {
    it('should render without errors ResultsBody', () => {
        const wrapper = shallow(<ResultsBody/>);
        expect(wrapper).toMatchSnapshot();
    });
});