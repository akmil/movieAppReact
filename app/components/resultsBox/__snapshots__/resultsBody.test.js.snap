// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ResultsBody should render without errors ResultsBody 1`] = `
<div
  className="results-list-box"
>
  <ul
    className="results-list"
  >
    <ResultsBodyItem
      0="b"
      1="o"
      2="o"
      key="0"
      onClick={[Function]}
    />
  </ul>
</div>
`;
